### PR TITLE
docs(offline-mode): Add offline mode docs for python and java

### PR DESCRIPTION
**Pull request by matthewelwell**
## Changes

Adds the framework for the docs on offline mode, as well as adding the specifics for python and Java. 

## How did you test this code?

Ran docusaurus locally and confirmed the docs looked correct.

### CODE REVIEW

File: docs/docs/clients/server-side.md [hunk 1]
```diff
@@ -749,6 +749,66 @@ client_configuration = Flagsmith.Client.new(environment_key: "MY_SDK_KEY", defau
 </TabItem>
 </Tabs>
 
+### Using an Offline Handler
+
+:::info
+
+Offline handlers are still in active development for some SDKs.
```
Comment (dabeeeenster) [inline docs/docs/clients/server-side.md]:
```suggestion
Offline handlers are still in active development. We are building them for all our SDK's; those that are production ready are listed below. 
```

File: docs/docs/clients/server-side.md [hunk 2]
```diff
@@ -804,6 +864,19 @@ Evaluation mode. Please see [caching](#caching) below.
 </TabItem>
 </Tabs>
 
+### Offline Mode
+
+:::info
+
+Offline mode is still in active development for some SDKs.
```
Comment (dabeeeenster) [inline docs/docs/clients/server-side.md]: Do we have a list somewhere to link to?

**Review by dabeeeenster — COMMENTED** [review]
Made some comments

**Comment by matthewelwell** [comment]
> Made some comments

Updated. Note that I've only added the docs for python and java, although more SDKs are complete. I am going to ask the rest of the team to add the docs for the relevant SDKs.

**Review by dabeeeenster — APPROVED** [review]

**Pull request merged**